pkg/util/maven/maven_command.go

```diff
@@ -74,6 +74,14 @@ func (c *Command) Do(ctx context.Context) error {
 		args = append(args, "--settings", settingsPath)
 	}
 
+	if cpuQuota, ok := os.LookupEnv("CPU_QUOTA"); ok {
+		args = append(args, "--define", "org.apache.camel.container.cpus="+cpuQuota)
+	}
+
+	if cpuQuota, ok := os.LookupEnv("CPU_QUOTA"); ok {
```

Comment:
Following #2837, I think the `GOMAXPROCS` environment variable can be used here.